go/stellar/inflation.go

```diff
@@ -9,8 +11,30 @@ import (
 // https://pool.lumenaut.net/
 const lumenautPoolAccountID = stellar1.AccountID("GCCD6AJOYZCUAQLX32ZJF2MKFFAUJ53PVCFQI3RHWKL3V47QYE2BNAUT")
 
+var predefinedInflationDestinations = [...]stellar1.PredefinedInflationDestination{
+	stellar1.PredefinedInflationDestination{
+		Tag:         stellar1.InflationDestinationTag("lumenaut"),
+		Name:        "Lumenaut",
+		Recommended: true,
+		AccountID:   stellar1.AccountID("GCCD6AJOYZCUAQLX32ZJF2MKFFAUJ53PVCFQI3RHWKL3V47QYE2BNAUT"),
+		Url:         "https://pool.lumenaut.net/",
+	},
+	stellar1.PredefinedInflationDestination{
+		Tag:       stellar1.InflationDestinationTag("sdf"),
+		Name:      "Stellar Developer Foundation",
+		AccountID: stellar1.AccountID("GDWNY2POLGK65VVKIH5KQSH7VWLKRTQ5M6ADLJAYC2UEHEBEARCZJWWI"),
+		Url:       "https://www.stellar.org",
+	},
+}
```

Comment:
I thought these were going to be on the server?  So we can change them without a client release.

Reply:
yes this needs to be server driven